lib/puppet/pops/parser/locator.rb

```diff
@@ -43,10 +43,6 @@ def offset_on_line(offset)
   def char_offset(byte_offset)
   end
 
-  # Returns the length measured in number of characters from the given start and end reported offset
-  def char_length(offset, end_offset)
-  end
-
   # Returns the length measured in number of characters from the given start and end byte offseta
```

Comment:
Could you remove the "a" at the end of the comment on the version we are keeping please

Reply:
Done, thanks!